cogs/core.py

```diff
@@ -249,20 +249,28 @@ async def acloseall(self, ctx, *, reason: str = None):
         usage="blacklist <member>",
         aliases=["block"],
     )
-    async def blacklist(self, ctx, *, member: MemberConverter):
+    async def blacklist(self, ctx, *, member: MemberConverter, reason: str = None):
         blacklist = (await tools.get_data(self.bot, ctx.guild.id))[9]
         if member.id in blacklist:
             await ctx.send(ErrorEmbed("The user is already blacklisted."))
             return
 
         async with self.bot.pool.acquire() as conn:
-            await conn.execute(
+            if len(reason) == 0:
```

Comment:
Should check if reason is None. Also, indentation is missing

Reply:
Fixed in #115